compiler/luci/pass/src/ConvertNCHWToNHWCPass.cpp

```diff
@@ -369,6 +369,26 @@ bool is_NCHW_with_const(const luci::CircleAdd *node, luci::CircleNode *&pred_nod
     return false;
 }
 
+template <class T> bool convert_Relu(T *node)
```

Comment:
How about naming this `convert_single_input_operator` and including Neg too?

`node->features()` in the function body may be replaced with `node->args(0)`.

Reply:
how about shorter `convert_unary` ?

Reply:
for future, I'll update this method as `convert_unary_features`

Reply:
> We cannot use node->args(0) as setter. so cannot merge with Neg

Oh I see. Thank you.